src/alice.ts

```diff
@@ -81,6 +83,46 @@ export class Alice implements IAlice {
     };
   }
 
+  public listen(
```

Comment:
This function is very long and complex, and gonna grow up if we add https support here. It will be great to split it to several methods. Or create separate class something like AliceHttpServer, move all this logic into it, and just call start\stop methods in alice.ts.

Reply:
I think https support is not in out interest yet.
Developers should better use `nginx` configuration or CloudFlare for `https`.

But it's rather nice idea to kick out server from main package to make it zen, as it was before.

Reply:
moved to a signle file